app/src/main/java/com/simplecity/amp_library/ui/presenters/PlayerPresenter.java

```diff
@@ -124,11 +129,15 @@ public void bindView(@NonNull PlayerView view) {
     }
 
 
-    private void refreshCurrentTimeText(long playbackTime) {
+    private void refreshTimeText(long playbackTime) {
```

Comment:
There appears to be an issue when a track is playing, and you skip tracks, the remaining time flickers to an incorrect value. This is likely because `totalTime` has not yet been updated to the new song's duration, but the current time is reset to zero, so we momentarily display the wrong value.

This can be resolved by using `MusicUtils.getDuration()` instead of referencing the (out of date) member variable `totalTime`. This change would then make the `totalTime` variable redundant, so you can remove it.